strongbox-testing/strongbox-testing-core/src/main/java/org/carlspring/strongbox/testing/AssignedPorts.java

```diff
@@ -40,7 +40,7 @@ public void initializePorts()
                 final int port = Integer.parseInt(System.getProperty(portKey));
                 ports.put(portKey, port);
 
-                logger.debug("Discovered port '" + portKey + "' = '" + port + "'.");
+                logger.debug("Discovered port '{}' = '{}'.", portKey, port);
```

Comment:
@fuss86 , @sbespalov ,

Should we perhaps have a follow-up task to drop this class and replace it with Spring `@Value`-s, or maybe there's an even better way now?